packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -363,6 +363,12 @@ export const gliner = (model: ModelData): string[] => [
 model = GLiNER.from_pretrained("${model.id}")`,
 ];
 
+
+export const htrflow = (model: ModelData): string[] => [
```

Comment:
We push to make the model snippets more complete, it might make sense to have a more complete python snippet like here: https://ai-riksarkivet.github.io/htrflow/latest/getting_started/pipeline.html#nested-segmentation

Based on the model checkpoints you could customise this snippet per task.

Reply:
Hi, I'm just how we should document it, since we are actually trying to import from different framework under the hood.. see snippet below as an example:

```py
from htrflow.pipeline.steps import Segmentation
from htrflow.pipeline.pipeline import Pipeline
from htrflow.volume.volume import Collection
from htrflow.models.ultralytics.yolo import YOLO

collection = Collection.from_directory("./")

# Create additional steps
segmentation_step = Segmentation(
    model_class=YOLO,
    model_kwargs={'model': 'Riksarkivet/yolov9-regions-1'},
    generation_kwargs={}
)

# Create the pipeline with multiple steps
pipeline = Pipeline(
    [
        segmentation_step,
 
    ]
)

pipeline.run(collection)
```

Reply:
Do we need to unify this or what do you think?

Reply:
The snippet in the latest commit looks good to me! 👍